src/metabase/models/card.clj

```diff
@@ -34,6 +34,39 @@
   [{:keys [id]}]
   (db/count 'DashboardCard, :card_id id))
 
+(defn dashboard-cards
+  "Return the corresponding dashboard cards for this Card"
+  {:hydrate :dashboard_cards}
+  [{:keys [id]}]
+  (db/select 'DashboardCard, :card_id id))
+
+(defn multi-cards
```

Comment:
I don't think there's a reason to use hydration if the call sites are

`(:multi_cards (hydrate card :multi_cards))`

Probably easier to just make it a regular function.

Reply:
crap, I didn't think of that. this does seem to give the expected error when there's two dashboard cards with multis with respect to the same card

how do you feel about an interface change to `render` multi?